miniapp/miniapp_recipes.cpp

```diff
@@ -134,6 +134,7 @@ class basic_cell_recipe: public recipe {
         std::exponential_distribution<float> delay_dist(delay_distribution_param_);
         float delay = param_.min_connection_delay_ms + delay_dist(rng);
         float weight = param_.syn_weight_per_cell/param_.num_synapses;
+        // not a full cell connection: domain must be filled by caller
```

Comment:
Actually, needs to be removed. In an earlier version, cell_connection had a domain member which was set to (unsigned) -1, but now that's gone, so the comment is not only confusing, but wrong.